pkg/log/main.go

```diff
@@ -41,7 +127,7 @@ func (l *Log) Inception() *event.Event {
 func (l *Log) Current() *event.Event {
 	sort.Sort(BySequence(l.Events))
 
-	return l.Events[len(l.Events)-1]
+	return l.Events[len(l.Events)-1].Event
 }
```

Comment:
to avoid a panic, should probably do a boundary check here

Reply:
Yes - there is nothing stopping you from calling `current` on an empty log